hub/test/src/@types/tape-promise/index.d.ts

```diff
@@ -0,0 +1,220 @@
+declare module 'tape-promise/tape' {
```

Comment:
Yeah, we actually just need to get this committed to https://github.com/DefinitelyTyped/DefinitelyTyped and eventually this will be a type package dependency. Its on my TODO list.